docker/Base/Dockerfile

```diff
@@ -0,0 +1,110 @@
+ARG CUDA_INT=113
+
+FROM nvidia/cuda:11.3.0-cudnn8-devel-ubuntu20.04 AS cuda-113
+FROM nvidia/cuda:10.2-cudnn8-devel-ubuntu18.04 AS cuda-102
+
+FROM cuda-${CUDA_INT} AS final
+
+ARG TORCH_VERSION=1.10.0
+ARG TORCHVISION_VERSION=0.11.0
+
+# important dependencies
+ARG OPENCV_VERSION==4.5.4.60
+ARG PPLCV_VERSION=0.7.0
+
+# backends
+ARG ONNXRUNTIME_VERSION=1.8.1
+ARG PPLNN_VERSION=0.8.1
+ARG NCNN_VERSION=20221128
+ARG TENSORRT_VERSION=8.2.3.0
+# tensorrt tar file url
+ARG TENSORRT_URL
+
+USER root
+WORKDIR /root/workspace
+
+ENV DEBIAN_FRONTEND=nointeractive
+ENV FORCE_CUDA="1"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        apt-utils \
+        ca-certificates \
+        gcc-7 \
+        g++-7 \
+        git \
+        vim \
+        wget \
+        libopencv-dev \
```

Comment:
Will this docker image be used for prebuilt package? If so, may use {OPENCV_VERSION} instead of `libopencv-dev`

Reply:
Actually, this image is not for prebuild.